lib/cusparse/generic.jl

```diff
@@ -1,6 +1,7 @@
 # generic APIs
 
-export vv!, sv!, sm!, gemm, gemm!
+export sparsetodense, densetosparse, gather!, scatter!, axpby!, rot!
```

Comment:
If we export this, I'd use `sparse_to_dense` and `dense_to_sparse`. Or maybe `to_dense` or even `dense` if we can disambiguate it sufficiently based on the arguments? Although that may be confusing for `sparse` (given that there already is a function with specific meaning).

Reply:
We don't need to export `sparsetodense`, `densetosparse`.
We use CuMatrix(A), `CuSparseMatriCSC(A)`, `CuSparseMatriCSR(A)`, `CuSparseMatriCOO(A)` in practice.
I will only export the other functions.